package.json

```diff
@@ -24,12 +30,15 @@
   },
   "homepage": "https://github.com/ppremk/lfs-warning#readme",
   "dependencies": {
-    "@actions/core": "^1.2.6",
-    "@actions/github": "^2.1.1"
+    "@actions/core": "^1.4.0",
+    "@actions/github": "^5.0.0",
+    "esbuild": "^0.12.17"
```

Comment:
@naseemkullah should we move this to devDependencies?

Reply:
true! reverted to use ncc as per github's internal recommendation